ci/ci_setup.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+##
+# Note this setup needs a system ruby to be available, this can not
+# be done here as is higly system dependant.
+##
+
+#squid proxy work, so if there is a proxy it can be cached.
+sed -i.bak 's/https:/http:/' Gemfile
+
+# Clean up some  possible stale directories
+rm -rf vendor       # make sure there are no vendorized dependencies
+rm -rf spec/reports # no stale spec reports from previous executions
+
+# Setup the environment
+rake bootstrap # Bootstrap your logstash instance
```

Comment:
what I meant is that calling `rake test:install-all` or `rake test:install-core` or rake test:install-defaults` will implicitly call `bootstrap`. So is `bootstrap` required here?